ssc/cmod_communitysolar.cpp

```diff
@@ -606,8 +606,6 @@ static var_info _cm_vtab_communitysolar[] = {
     /* PPA revenue not applicable to community solar, may need to be restored later
     { SSC_OUTPUT,       SSC_ARRAY,      "cf_ppa_price",                           "PPA price",                     "cents/kWh",      "",                      "Cash Flow Revenues",             "*",                      "LENGTH_EQUAL=cf_length",                             "" },
     { SSC_OUTPUT,       SSC_ARRAY,      "cf_energy_value",                        "PPA revenue net",                     "$",      "",                      "Cash Flow Revenues",             "*",                      "LENGTH_EQUAL=cf_length",                             "" },
```

Comment:
I am confused based on the email chain... are we keeping "net" here or does that go away when @cpaulgilman merges his changes?

Reply:
We can remove net, but this is in commented out code, so it won't change anything. Should I remove net or just remove the whole block?

Reply:
Good point... we should be consistent and removing commented code - only keep if necessary or informative. Would you say this is clarifying or just a historic artifact? (Useful for debugging in the future?)